src/store/reducers/trace.ts

```diff
@@ -0,0 +1,28 @@
+import type {BaseQueryFn, EndpointBuilder} from '@reduxjs/toolkit/query';
+
+import {api} from './api';
+
+interface CheckTraceParams {
+    url?: string;
+}
+
+function endpoints(build: EndpointBuilder<BaseQueryFn, string, string>) {
+    return {
+        checkTrace: build.query<{}, CheckTraceParams>({
+            queryFn: async ({url}, {signal}) => {
+                try {
+                    if (!url) {
+                        throw new Error('no tracecheck url provided');
+                    }
+                    const response = await window.api.checkTrace({url}, {signal});
+                    return {data: response};
+                } catch (error) {
+                    return {error: error};
+                }
+            },
+            providesTags: ['All'],
```

Comment:
Our refresh control refreshes data by `All` tag. You don't need to refresh this query, this tag is not needed